plugins/trace_api_plugin/include/eosio/trace_api/extract_util.hpp

```diff
@@ -4,17 +4,29 @@
 
 namespace eosio { namespace trace_api {
 
-/// Used by to_transaction_trace  for creation of action_trace_v0 or action_trace_v1
+/// Used by to_transaction_trace  for creation of action_trace_v0, action_trace_v1
+/// or action_trace_v2
 template<typename ActionTrace>
 inline ActionTrace to_action_trace( const chain::action_trace& at ) {
    ActionTrace r;
    r.receiver = at.receiver;
    r.account = at.act.account;
    r.action = at.act.name;
    r.data = at.act.data;
-   if constexpr(std::is_same_v<ActionTrace, action_trace_v1>) {
+   if constexpr(std::is_same_v<ActionTrace, action_trace_v1> ||
+                std::is_same_v<ActionTrace, action_trace_v2>) {
       r.return_value = at.return_value;
    }
+   if constexpr(std::is_same_v<ActionTrace, action_trace_v2>) {
+      r.call_traces.reserve( at.call_traces.size());
+      for( const auto& t : at.call_traces ) {
+         r.call_traces.emplace_back( call_trace_v0{t.call_ordinal,
+                                                   t.sender_ordinal,
```

Comment:
Not sure we need either ordinal. We don't have `action_trace`: `action_ordinal`, `creator_action_ordinal`, or `closest_unnotified_ancestor_action_ordinal`.

Reply:
Maybe keep the `call_ordinal` so we have a way to identify a sync call? `action_trace` has global sequence.

Reply:
If you think `sender_ordinal` is also useful, not opposed to adding it.

Reply:
Will keep `sender_ordinal` for now as it helps identifying call hierarchy.